docs/relabund.rst

```diff
@@ -0,0 +1,66 @@
+.. _RelAbund:
+
+**************
+RelAbund Class
+**************
+
+.. index:: RelAbund
+
+Notebooks
+=========
+
+.. toctree::
+   :maxdepth: 1
+
+   Simple Examples <RelAbund>
+
+Overview
+========
+
+This class packages the relative abundances of an object, typically
+an AbsSystem.
+
+Instantation
+============
+
+To begin, make a new class instance::
+
+   XY = RelAbund()
+   Loading abundances from Asplund2009
+
+Then load data into the data dict.  Here is an example::
+
+   XY._data = {6: dict(flag=1, XH=-1., sigXH=0.2, sig=0.05),
+            8: dict(flag=2, XH=-1.4, sigXH=0.25, sig=0.05),
+            14: dict(flag=1, XH=-1.1, sigXH=0.25, sig=0.05),
+            26: dict(flag=1, XH=-1.4, sigXH=0.25, sig=0.05),
+            32: dict(flag=3, XH=-0.8, sigXH=0.25, sig=0.05),
+            }
+
```

Comment:
What do the flag values mean? They should be described here. Ideally the flag values could be short strings that suggest what they are. e.g. 'lo', 'hi' (or whatever) for lower and upper limits. (But I realise this means making changes elsewhere)